plataform_front_end/src/components/loader/loader.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./loader.css";
+
+const Loading = () => {
+    return <div className="homer"><p>Homer</p></div>
+}
+
+export default Loading;
```

Comment:
'No new line at end of file' alert.

Reply:
You can use [EditorConfig](https://editorconfig.org/) for this in vscode and many others code editors, just make sure to enable the `insert_final_newline` option, vscode will handle it automatically for you if you have the `Format On Save` option enabled on workspace settings. You can also take a look on [Prettier](https://prettier.io/) which is more advanced, but can format your code automatically.